cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AbstractMonitorHandler.java

```diff
@@ -268,4 +271,63 @@ public void getLatestRestartServiceInstanceStatus(HttpRequest request, HttpRespo
                            String.format("No restart instances request found or %s", serviceName));
     }
   }
+
+  public void updateServiceLogLevels(HttpRequest request, HttpResponder responder, String serviceName) {
+    if (!serviceManagementMap.containsKey(serviceName)) {
+      responder.sendString(HttpResponseStatus.NOT_FOUND, String.format("Invalid service name %s", serviceName));
+      return;
+    }
+
+    MasterServiceManager masterServiceManager = serviceManagementMap.get(serviceName);
+    if (!masterServiceManager.isServiceEnabled()) {
+      responder.sendString(HttpResponseStatus.FORBIDDEN,
```

Comment:
Similar here. Should throw exception. Currently we don't have one for `Forbidden`. Please create one in cdap-common.